scripts/us_epa/superfund/process_sites_fundingStatus.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script proceses data files for fundingStatus of EPA NPL Superfund sites
+
+About this script:
+This script processes data from:
+- ./data/Superfund National Priorities List (NPL) Sites with Status Information.csv
+  This file lists all superfund sites on the NPL with a note on the site's status on the NPL. This file also provides different dates when the NPL status of a site changed. This dataset is exported from the data view in  https://epa.maps.arcgis.com/apps/webappviewer/index.html?id=33cebcdfdd1b4c3a8b51d416956c41f1
```

Comment:
I have updated the directory structure a bit and have added more information about the dataset, how it can be downloaded in the README file in each sub-directory. The README in the directory's root serves as a navigation documentation.